staging/publishing/rules.yaml

```diff
@@ -2007,6 +2007,17 @@ rules:
     source:
       branch: master
       dir: staging/src/k8s.io/dynamic-resource-allocation
+- destination: noderesourcetopology-api
+  branches:
+  - name: master
+    dependencies:
+    - repository: code-generator
+      branch: master
+      dir: staging/src/k8s.io/noderesourcetopology-api
```

Comment:
I don't know the rules.yaml well, but this "dir" seems wrong to me - please confrim @dims?

Reply:
@thockin we already landed something similar last week (see `dynamic-resource-allocation`), if this new `destination` looks like that then we are good!

Reply:
heading out for an errand, will let @swatisehgal confirm that it does look like the other one.

Reply:
(also this will land when `master` reopens for 1.27)